DISCOVER/04_venue_selection.md

```diff
@@ -44,8 +44,8 @@
 - [Improving the Color Accessibility for Color Blind Users](https://www.smashingmagazine.com/2016/06/improving-color-accessibility-for-color-blind-users/)
 - [Accessibility, sensory friendliness and neurodiversity friendliness at events](https://gist.github.com/katsel/b8fbd1a35734708d5e654bb5153a5aa3) via [@katsel](https://gist.github.com/katsel)
 - [Captioning vs Sign Language – A No Brainer?](https://therebuttal2.com/2016/01/28/captioning-vs-sign-language-a-no-brainer/)
-- [A Note From Your Colleagues With Hearing Loss: Just Use a Microphone Already](https://www.chronicle.com/article/A-Note-From-Your-Colleagues/245916)
-- [What is a hearing loop?](https://www.hearinglink.org/living/loops-equipment/hearing-loops/what-is-a-hearing-loop/)
+<!-- Citation: "A Note From Your Colleagues With Hearing Loss: Just Use a Microphone Already," The Chronicle of Higher Education, March 13, 2019. Available at: https://www.chronicle.com/article/A-Note-From-Your-Colleagues/245916 (subscription may be required for full access). -->
```

Comment:
Same comment as above, would recommend we use[ markdown footnotes](https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/) rather than HTML comments.